corehq/warehouse/migrations/0032_auto_20190917_1542.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations, models
+
+TABLE_NAME = 'warehouse_synclogstagingtable'
+INDEX_NAME = 'warehouse_s_user_id_8bcfbf_idx'
+COLUMNS = ['user_id']
+
+
+CREATE_INDEX_SQL = "CREATE INDEX CONCURRENTLY IF NOT EXISTS {} ON {} ({})".format(
+    INDEX_NAME, TABLE_NAME, ','.join(COLUMNS)
+)
+DROP_INDEX_SQL = "DROP INDEX CONCURRENTLY {}".format(INDEX_NAME)
```

Comment:
Get excited for concurrent indexes natively supported https://docs.djangoproject.com/en/3.0/ref/contrib/postgres/operations/#index-concurrent-operations